config/initializers/elasticsearch.rb

```diff
@@ -1,9 +1,37 @@
-config = {
-  host: ENV.fetch("ES_HOST", "http://127.0.0.1:9200"),
-  transport_options: {
-    request: { timeout: 5 }
+require 'base64'
+
+vcap_services = ENV.fetch("VCAP_SERVICES", nil)
+
+if vcap_services
+  # we are running on PaaS
+  vcap = JSON.parse(vcap_services)
+
+  es_server = vcap['elasticsearch'][0]['credentials']['uri'].chomp("/")
+
+  es_cert = Base64.decode64(vcap['elasticsearch'][0]['credentials']['ca_certificate_base64'])
+  es_cert_file = File.new("/tmp/out.pem", "w")
+  es_cert_file.puts(es_cert)
+  es_cert_file.close
+
+  puts es_server
+  puts es_cert_file.path
+
+  config = {
+    host: es_server,
+    transport_options: {
+      request: { timeout: 5 },
+      ssl: { ca_file: es_cert_file.path }
+    }
   }
-}
+else
+  es_server = ENV.fetch("ES_HOST", "http://127.0.0.1:9200")
+  config = {
+    host: es_server,
+    transport_options: {
+      request: { timeout: 5 }
```

Comment:
Not sure if that's important or not, but I have seen the docs write:

`Elasticsearch::Client.new request_timeout: 5`

https://github.com/elastic/elasticsearch-ruby/tree/master/elasticsearch-transport#setting-timeouts